CONTRIBUTING.md

```diff
@@ -57,25 +57,39 @@ All changes must be code reviewed. Coding conventions and standards are explaine
 docs](https://git.k8s.io/community/contributors/devel). Expect reviewers to request that you
 avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
 
+## Releases
+
+Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) to track releases.
+
+- Minor versions CAN be planned and scheduled twice / year.
+- Patch versions CAN be planned and scheduled each month.
+- Code freeze is in effect 72 hours (3 days) before a release.
```

Comment:
How will the code freeze be communicated and enforced? Will the main branch still be open for merges during that time?

Reply:
Probably depends, if a release branch is created we can merge PRs in, otherwise we should probably wait

In general, I don't think we need to hold every single PR (that would create lots of noise), we can leave PRs without explicit approvals until the very last minute

Reply:
In terms of communication, we should communicate at least 1 week before the release at one of the community meetings, I'll make it explicit in here